doc/3/controllers/security/create-user/index.md

```diff
@@ -0,0 +1,54 @@
+---
+code: true
+type: page
+title: createUser
+description: Security:createUser
+---
+
+# createUser
+
+Creates a new user in Kuzzle.
+
+:::info
+There is a small delay between user creation and its availability in our search layer (usually a couple of seconds).
+That means that a user that was just created may not be returned by the `searchUsers` function at first.
+:::
+
+---
+
+## createUser(id, user, [options], [callback])
+
+| Arguments  | Type        | Description                                                                                                                  |
+| ---------- | ----------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| `id`       | string      | [Unique user identifier](/core/1/guides/essentials/user-authentication#kuzzle-user-identifier-kuid)                   |
+| `user`     | JSON Object | A plain JSON object representing the user (see below)                                                                        |
+| `options`  | string      | (Optional) Optional arguments                                                                                                |
+| `callback` | function    | Callback handling the response                                                                                               |
+| `refresh`  | string      | If set to `wait_for`, Kuzzle will wait the persistence layer to finish indexing |
+
+The `user` object to provide must have the following properties:
+
+- `content` (JSON object): user global properties
+  - This object must contain a `profileIds` properties, an array of strings listing the security [profiles](/core/1/guides/essentials/security#users-profiles-and-roles) to be attached to the new user
+  - Any other property will be copied as additional global user information
+- `credentials` (JSON object): a description of how the new user can identify themselves on Kuzzle
+  - Any number of credentials can be added, each one being an object with name equal to the [authentication strategy](/core/1/plugins/guides/strategies#exposing-authentication-strategies) used to authenticate the user, and with the login data as content.
+  - If this object is left empty, the user will be created in Kuzzle but the will not be able to login.
+
+---
+
+## Options
+
+| Filter     | Type    | Description                      
+| ---------- | ------- | ---------------------------------
+| `queuable` | boolean | Make this request queuable or not
```

Comment:
Is this still an available option in this version of the SDK? If so, the "new" description must be used:

`If true, queues the request during downtime, until connected to Kuzzle again`

Reply:
According to @alexandrebouthinon . it still exists

Reply:
![image](https://user-images.githubusercontent.com/4447392/74430832-f5bdc080-4e5d-11ea-9a0c-f3795e981658.png)

Reply:
The SDK PHP v4 is based on SDK JS v6 so https://docs.kuzzle.io/sdk/js/6/controllers/document/count/